utils/datastore/models.go

```diff
@@ -2,10 +2,59 @@ package datastore
 
 import (
 	"database/sql"
+	"database/sql/driver"
 	"encoding/json"
+	"errors"
 	"strings"
+
+	"github.com/lib/pq"
 )
 
+// Methods type is a string slice holding payments
```

Comment:
Comment pertains to the filename, would it be possible to file this under `payments/utils/datastore/payment_metadata.go` or something similar? The structure seemed confusing since I thought it would be closer to MVC (and it sort of is but far from beego-like)

Reply:
:+1:

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/816/commits/6276de6cc2ffc85e725c37c13ded51bd438902a1